{"name": "test_login_with_invalid_credentials", "status": "passed", "steps": [{"name": "Logging in with invalid credentials", "status": "passed", "steps": [{"name": "Logging response", "status": "passed", "attachments": [{"name": "Response Text", "source": "c60c328f-adbe-4e40-961c-59f2c5ff27b1-attachment.json", "type": "application/json"}, {"name": "Response Headers", "source": "01711a8d-659c-4c19-a2c0-9155bb3a1e4a-attachment.json", "type": "application/json"}], "start": 1719519456918, "stop": 1719519456919}], "start": 1719519455807, "stop": 1719519456919}], "start": 1719519455807, "stop": 1719519456919, "uuid": "1f7a8b37-b429-4d66-99df-19d5c6e94f7b", "historyId": "7071ba7437dcd01d6bcb4f3d93e3edfc", "testCaseId": "7071ba7437dcd01d6bcb4f3d93e3edfc", "fullName": "tests.test_user_operations#test_login_with_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "9008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}